Makefile

```diff
@@ -15,7 +15,7 @@ all_to_check=$(files_to_check) $(dirs_to_check)
 .PHONY: ci
 ci: deps checkgofmt vet staticcheck ineffassign predeclared test
 
-INSTALLTOOL = GO111MODULE=off go get
+INSTALLTOOL = GO111MODULE=on go get
```

Comment:
Given our conversation earlier, I thought it was also okay to go ahead and flip this to on. Just calling `make` was failing for me (include `make release`), which prompted me to change it.

Reply:
You actually want `go install` with modules. A `go get` with modules will actually update to the latest version. (I had to recently change this in the grpcurl and grpcui `Makefile`s too.)

Reply:
Ah, good to know!